deploy/helm/values.yaml

```diff
@@ -26,6 +26,7 @@ selinuxdImage:
 
 enableSelinux: false
 enableLogEnricher: false
+enableJsonEnricher: false
```

Comment:
if there's going to be a configurable log interval period in the future, it almost makes sense to me to have this be a pointer to a jsonEnricher object that has an interval embedded. if the object is nil, no enrichment. if the interval is unset (0) then it defaults to 60 seconds, otherwise it uses the value of the interval